pkgs/by-name/ur/uradvd/package.nix

```diff
@@ -2,17 +2,25 @@
   stdenv,
   lib,
   fetchFromGitHub,
+  git,
+  testers,
+  uradvd,
 }:
 
-stdenv.mkDerivation {
+stdenv.mkDerivation (finalAttrs: {
   pname = "uradvd";
-  version = "0-unstable-2025-08-16";
+  version = "r26-1e64364d";
+
+  nativeBuildInputs = [
```

Comment:
Nit: `nativeBuildInputs` should go below `src`.

Reply:
resolved in 32cf96dd3ba32c342e849619519910d74973ecec